core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -60,9 +60,13 @@ export class DotContainerPropertiesComponent implements OnInit {
                 });
 
                 this.addContainerFormControl(containerStructures);
+                if (this.form.value?.identifier)
+                    this.store.updateOriginalFormState(this.form.value);
```

Comment:
We try to follow a more concise code styling, so please use `{}`

Reply:
Noted